### DISCUSSION TITLE
Restrict admin panel by IP

### DISCUSSION THREAD

**Discussion by ArtDepartmentMJ** [verbb/knock-knock#72] [General]
Can this be used to only restrict the admin panel by IP addresses and if so what would that config look like?

**Comment by engram-design** [comment]
You can if you like! There's an "Enabled CP Protection" you can enable to protect the control panel, not the front-end. You can then set "Allow IPs" to use on the IPs you nominate to have access. Users who try to access the control panel will be met with the Knock Knock login challenge.

You can of course change that template to be whatever you like. Personally, I don't feel like adding the Knock Knock login over the top of the Craft login to be actually useful. So you might prefer to override the template to show a generate "you are not authorized" message instead.
